test/controllers/amends/AmendExportedByAnotherBusinessControllerSpec.scala

```diff
@@ -16,144 +16,167 @@
 
 package controllers.amends
 
-import base.SpecBase
+import base.utils.JourneyActionAnswer
+import cacheables.AmendObligationCacheable
 import connectors.CacheConnector
+import controllers.BetterMockActionSyntax
+import controllers.actions.JourneyAction
 import forms.amends.AmendExportedByAnotherBusinessFormProvider
-import models.Mode.NormalMode
 import models.UserAnswers
-import navigation.{FakeNavigator, Navigator}
+import models.UserAnswers.SaveUserAnswerFunc
+import models.requests.DataRequest
+import models.returns.TaxReturnObligation
+import org.mockito.Answers
+import org.mockito.ArgumentMatchers.{eq => meq}
 import org.mockito.ArgumentMatchersSugar.any
-import org.mockito.MockitoSugar.when
+import org.mockito.MockitoSugar.{reset, verify, when}
+import org.scalatest.BeforeAndAfterEach
 import org.scalatestplus.mockito.MockitoSugar
+import org.scalatestplus.play.PlaySpec
 import pages.amends.AmendExportedByAnotherBusinessPage
-import play.api.inject.bind
-import play.api.mvc.Call
+import play.api.data.Form
+import play.api.i18n.MessagesApi
+import play.api.mvc.{Action, AnyContent}
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
+import play.twirl.api.Html
+import queries.{Gettable, Settable}
+import support.AmendExportedData
 import views.html.amends.AmendExportedByAnotherBusinessView
 
+import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
+import scala.util.Try
+
+class AmendExportedByAnotherBusinessControllerSpec
+  extends PlaySpec
+    with JourneyActionAnswer
+    with MockitoSugar
+    with AmendExportedData
+    with BeforeAndAfterEach {
+
+  private val dataRequest = mock[DataRequest[AnyContent]](Answers.RETURNS_DEEP_STUBS)
+  private val userAnswer = mock[UserAnswers]
```

Comment:
as datarequest is mocked, this mock shouldnt be needed

Reply:
yes I agree